src/Entity/Content.php

```diff
@@ -458,4 +459,42 @@ public function __call(string $name, array $arguments = []): Field
             throw new \RuntimeException(sprintf('Invalid field name or method call on %s: %s', $this->__toString(), $name));
         }
     }
+
+    public function serialize(): string
```

Comment:
use jsonSerialize() from ContentExtrasTrait

Reply:
This is not used now.